docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -50,7 +87,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "To show pulse-efficient transpilation, we will pick a backend that supports OpenPulse extension. Note that the current implementation only supports backend that has `ECRGate` as the two-qubit basis gate."
+    "To perform pulse-efficient transpilation, choose a backend from the Eagle or Falcon family, because only these processor families support running OpenPulse circuits.  Note that the implementation shown in this notebook only supports backends that has `ECRGate` as the two-qubit basis gate."
```

Comment:
a minor typo here, "only supports backends that has `ECRGate` as the two-qubit basis gate" -> "only supports backends that have `ECRGate` as the two-qubit basis gate"